src/main/java/com/potionsbyfreya/songr/models/AlbumRepository.java

```diff
@@ -3,4 +3,5 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface AlbumRepository  extends JpaRepository<Album, Long> {
+    public Album findByTitleOfAlbum(Album title);

```

Comment:
This method should just be called `findByTitle`. Spring know's `findBy` and you can add the name of an objects property and Spring knows to find by the property. If you add the `OfAlbum` it might confuse Spring because there's no `titleOfAlbum` property on albums.